Could not create plugin of type 'AndroidMavenPlugin'

Hello,
upon building your app in Android Studio 2.2 Beta 2 with gradle plugin **com.android.tools.build:gradle:2.2.0-beta2**, I get the following error:
> Error:FAILURE: Build failed with an exception.
* Where:
Script '/Users/igorganapolsky/workspace/git/google/easypermissions/easypermissions/maven.gradle' line: 1
* What went wrong:
A problem occurred evaluating script.
> Failed to apply plugin [id 'com.github.dcendents.android-maven']
   > Could not create plugin of type 'AndroidMavenPlugin'.

If I debug the build from commandline, I see the following info:
> Caused by: org.gradle.internal.service.UnknownServiceException: No service of type Factory<LoggingManagerInternal> available in ProjectScopeServices.


Any ideas?

Thanks,
Igor

--- a/build.gradle
+++ b/build.gradle
@@ -5,9 +5,9 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.1'
 
-        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1'
     }
 }

--- a/easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java
+++ b/easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java
@@ -294,7 +294,7 @@ public class EasyPermissions {
                     // Method must be void so that we can invoke it
                     if (method.getParameterTypes().length > 0) {
                         throw new RuntimeException(
-                                "Cannot execute non-void method " + method.getName());
+                                "Cannot execute method " + method.getName() + " because it is non-void method and/or has input parameters.");
                     }
 
                     try {

